traitsui/tests/editors/test_animatedGIF_editor.py

```diff
@@ -39,7 +40,7 @@ def test_animated_gif_editor(self):
         view = View(
             Item('gif_file', editor=AnimatedGIFEditor(playing='playing')),
         )
-        
+
```

Comment:
This is my IDE's doing, not me.
(The change complies with flake8 so I am keeping it.)